### PR TITLE
Webapp status

### PR DESCRIPTION
**Pull request by danieljbae**
## What user stories have been completed?
1.  A user can register/create an account with their @colorado.edu email
2. Update account (only if a user is logged in)
3. Create/Update/Delete a Project post

## Summary main files that have been changed:
- main.css and layout.html
- Routes.py: defines route location, renders pages, collects user data from forms
- forms.py: defines fieldset on forms

### `main.css` and `layout.html` and other `.html` files
Here are a few of the changes I made
- User profile pic icon on the navbar, as a drop-down menu 
- Dark mode CSS selectors for web layout
- Dark mode CSS selectors for form pop-ups

TLDR: Dark mode!!!

### `routes.py`
This file serves the following purpose: 
- defines route location, and what GETs and POSTs do 
- on GET requests: renders .html pages, default fields, etc.
- on POST requests: collects user data from forms, to validate w/ database or creates an entry in database, etc.

### `forms.py` added the following forms
- update account form
- project creation forms 

image
image

### CODE REVIEW

**Comment by danieljbae** [comment]
## Remaining work 
Just my estimation, let me know if you guys want to prioritize other things

### 1. Features or user stories
- "Join" button on projects
> Currently the "Join" button shows for projects that a user is not the author of. However, the actual functionality of the button: adding users as `project.memebers`, still needs to be completed. 
- "Show Group Members" button on projects

### 2. Deploy App via Heroku
- Flask has a tool called `blueprints`, which should help with deployment.
- I'm going to start working on this before I fly out to help get the ball rolling on this

### Low priority (in my opinion)
- View other people's profile
- Adding programming languages and careers to the user account page and registration form
- "View" button for programming languages and careers for projects. 
- etc.
